packages/cli-platform-ios/README.md

```diff
@@ -47,6 +47,10 @@ react-native run-ios --simulator "iPhone 14"
 
 [Deprecated] Explicitly set the scheme configuration to use default: 'Debug'.
 
+#### `--mode <string>`
+
+Explicitly set the scheme configuration to use. This option is case sensitive.
```

Comment:
Can we add example options to use, or default value?

Reply:
done ✅